code/auth/README.md

```diff
@@ -42,15 +42,15 @@ This repo utilizes a OpenShift's source-to-image (s2i) for customizing and layer
 ### Building this service
 Build requires an OpenShift cluster the new-build must point to a git repo or be pointing to a locally cloned branch. For secrets, branches and other options type `oc new-build -h`
 ```bash
-oc new-build redhat-sso-7/sso73-openshift:latest~https://github.com/dudash/openshift-microservices.git#develop \
+oc new-build redhat-sso73-openshift:latest~https://github.com/dudash/openshift-microservices.git#develop \
     --name=auth \
-    -l app=auth-sso73-x509
+    -l app=auth-sso73-x509 \
+    --context-dir=/code/auth
 ```
 
 ### Building a container image for this service
 As an alternative to building on OpenShift, you can use [s2i][4] to build this into a container image. For example:
 ```bash
-rm -rf node_modules
```

Comment:
I was having s2i build issues without cleaning up old node_modules before each build. Were you not seeing that?

Reply:
oh nevermind... this is the auth image, no node.js. :)